shishito/library/modules/runtime/shishito_support.py

```diff
@@ -7,84 +7,143 @@
 
 
 class ShishitoSupport(object):
-    def __init__(self):
-        # TODO this does not work without a runner
-        self.project_root = os.getcwd()
+
+    def __init__(self, cmd_args=None, project_root=None):
+        """ cmd_args added only if called from runner.
+        when called from tests cmd_args have to be added to pytest and taken from pytest.config
+        """
+
+        # parsed arguments from commad_line
+        self.args_config = {} if cmd_args is None else cmd_args
+
+        # called without cmd_args
+        # if True gid will use only pytest.config
+        # if False gid uses pytest.config, cmd_args and config files
+        self.used_in_test = True if cmd_args is None else False
+
+        if project_root is not None:
```

Comment:
better use positive conditions. `if project_root is None` ... and switch branches

btw can be project_root empty? because if you don't want to recognize empty and None then common pattern is using just `if project_root:`. But I don't know if this is that case